clang/docs/checkedc/Update-to-latest-LLVM-sources.md

```diff
@@ -46,6 +53,7 @@ that first.
 You can now branch your baseline branches to create a new master branch:
 
     git checkout -b updated-master
+    git config --local merge.renamelimit 26000
```

Comment:
Instead of hard-coding the number, can you add a command to determine the number that should be used for a given merge?  I think `git diff` has shown me a warning message with the number of potential renames in that particular diff; if you then set that as the limit, you are guaranteed that the limit will not cause any renames to be missed.

Reply:
Would the `git diff` command to check the necessary rename limit need to be run after attempting to run `git merge master`? Ideally the merge rename limit should be set before attempting to merge so that the merge works the first time.

Reply:
I think there is a way to check it before the merge, but I realized my second time around that it's probably more sensible to just set a number larger than anything you could ever need.  Was 26000 supposed to be such a number?  Maybe 99999999 or so would make the point clearer?